create-login-profile.js

```diff
@@ -36,6 +44,10 @@ function cliOpts() {
       type: "boolean",
       default: false,
     },
+
+    "interactive": {
+      describe: "Start in interactive mode!",
```

Comment:
we could have the default be false for consistency

Reply:
ok, will add